fabcar/java/pom.xml

```diff
@@ -28,7 +28,7 @@
       <dependency>
          <groupId>org.hyperledger.fabric</groupId>
          <artifactId>fabric-gateway-java</artifactId>
-         <version>1.4.1-SNAPSHOT</version>
+         <version>1.4.3</version>
```

Comment:
why would master use 1.4.3 rather than fabric-gateway-java 2.0.0?

Reply:
Well, we'll need to update the code for that upgrade. I just want to fix the CI for now. :-)